### ISSUE TITLE
Printing agenda width doesn't fit

### ISSUE THREAD

**Issue by hlnlwj** [JFXtras/jfxtras#67]
Hi,

When printing from the agenda, it seems that the width doesn't fit when using `AgendaWeekSkin` and `AgendaDaysFromDisplayedSkin`.
Indeed, when using both of these skins, the width of the agenda doesn't fit the page layout of the print job and the last day of the agenda isn't displayed as showed in the attached images (taken from the  JFXTras 8.0 Samples).

week view (saturday missing)
![week view (saturday missing)](https://cloud.githubusercontent.com/assets/9353824/17838031/c0eb6f02-67ba-11e6-80e3-994eb8181400.jpg)
dynamic view (8 days, saturday missing)
![dynamic view (8 days, saturday missing)](https://cloud.githubusercontent.com/assets/9353824/17838034/d8ffb3aa-67ba-11e6-80ec-fdb7ac162408.jpg)

**Comment by tbee** [comment]
Yeah. Printing in JavaFX was not quite finished when I wrote that code. Maybe now I have more control. I'll take a look.

**Comment by hlnlwj** [comment]
Thank you !
